### PR TITLE
Added a hackathon event.

### PR DESCRIPTION
**Pull request by udit-001**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ npm run dev
 
 ## Maintainer
 
-This project is maintainer by [geekyd](https://github.com/geekyd)
+This project is maintained by [geekyd](https://github.com/geekyd)
```
Comment (geekyd) [inline README.md]: These changes have already been made please update your branch.

File: static/data/2017/10/events.json [hunk 2]
```diff
@@ -161,4 +161,13 @@
     "website": "http://hackdtu.tech/",
     "logo": "http://hackdtu.tech/static/images/logo.png"
   }
+  {
+    "name": "Hack@BVP",
+    "start": "10/10/2017",
+    "end": "11/10/2017",
+    "location": "Bharati Vidyapeeth College of Engineering, GGSIPU",
```
Comment (geekyd) [inline static/data/2017/10/events.json]: Follow this format `City, Country`

File: static/data/2017/10/events.json [hunk 3]
```diff
@@ -161,4 +161,13 @@
     "website": "http://hackdtu.tech/",
     "logo": "http://hackdtu.tech/static/images/logo.png"
   }
```
Comment (geekyd) [inline static/data/2017/10/events.json]: Add a `,` here, this is a JSON we need it here.

Comment (udit-001) [reply static/data/2017/10/events.json]: Fixed it, sorry I'm kinda new to JSON!

File: static/data/2017/10/events.json [hunk 4]
```diff
@@ -161,4 +161,13 @@
...
+  {
+    "name": "Hack@BVP",
+    "start": "10/10/2017",
+    "end": "11/10/2017",
+    "location": "Bharati Vidyapeeth College of Engineering, Delhi, India",
```
Comment (udit-001) [inline static/data/2017/10/events.json]: Fixed the location according to the format!

File: static/data/2017/10/events.json (hunk 3 above)
Comment (geekyd) [reply static/data/2017/10/events.json]: The comma is still missing please add it.

File: static/data/2017/10/events.json [hunk 5]
```diff
@@ -160,5 +160,14 @@
     "type": "hackathon",
     "website": "http://hackdtu.tech/",
     "logo": "http://hackdtu.tech/static/images/logo.png"
+  },
```
Comment (udit-001) [inline static/data/2017/10/events.json]: I added the comma.

**Pull request merged**